docker-compose.yml

```diff
@@ -1,17 +1,46 @@
 version: '3.6'
 services:
-  license_finder:
-    build:
-      dockerfile: dockerfiles/Dockerfile.audit
-      context: .
+  mongodb:
+    image: mongo:latest
     volumes:
-      - ./:/scan
+      - "dbdata_mongodb:/data/db"
+    environment:
+      MONGO_INITDB_ROOT_USERNAME:
+      MONGO_INITDB_ROOT_PASSWORD:
+    restart: always
 
   test-management:
     build:
       dockerfile: dockerfiles/Dockerfile.tms
       context: .
+    depends_on:
+      - mongodb
     environment:
       BITBAR_ACCOUNT_MAX:
+      MONGODB_HOST: mongo
+      MONGODB_DB: test-reports
+      MONGODB_USER:
+      MONGODB_PASS:
+    links:
+      - "mongodb:mongo"
     ports:
       - "9340:9340"
```

Comment:
So just to confirm the deployment architecture - the test management service is run on one of the Macs, exposing port 9340 to the internet. Is it worth having a shared token for reading and writing to this service?